### PR TITLE
Correct bld.bat

### PR DESCRIPTION
**Pull request by danpetry**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

- the "if" from "if not exist" wasn't present, resulting in a script failure
- the files in the include\ directory weren't being moved because windows doesn't move recursively by default
- looking in the built package without changes from the conda-forge channel shows that it's empty

### CODE REVIEW

**Comment by danpetry** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
cc @msarahan (to review Windows changes)

**Comment by danpetry** [comment]
@conda-forge-admin, please rerender

**Review by msarahan — APPROVED** [review]

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+if not exist %LIBRARY_INC%\CL mkdir %LIBRARY_INC%\CL
+move include\CL\* %LIBRARY_INC%\CL
 
- not exist %LIBRARY_PREFIX%\etc mkdir %LIBRARY_PREFIX%\etc
- not exist %LIBRARY_PREFIX%\etc\OpenCL mkdir %LIBRARY_PREFIX%\etc\OpenCL
- not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
- type nul >> %LIBRARY_PREFIX%\etc\OpenCL\vendors\cuda.icd
+if not exist %LIBRARY_PREFIX%\etc mkdir %LIBRARY_PREFIX%\etc
+if not exist %LIBRARY_PREFIX%\etc\OpenCL mkdir %LIBRARY_PREFIX%\etc\OpenCL
+if not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
```
Comment (msarahan) [inline recipe/bld.bat:9]: Nit: Windows mkdir is recursive, so you only need one of these. https://stackoverflow.com/a/905237

Not a required change for merge, just pointing out.

Comment (msarahan) [inline recipe/bld.bat:9]:
```suggestion
if not exist %LIBRARY_PREFIX%\etc\OpenCL\vendors mkdir %LIBRARY_PREFIX%\etc\OpenCL\vendors
```

Nit: Windows mkdir is recursive, so you only need one of these. https://stackoverflow.com/a/905237

Not a required change for merge, just pointing out.

**Comment by jakirkham** [comment]
Thank you both! 🙏

**Pull request merged**